src/PassportServiceProvider.php

```diff
@@ -50,6 +50,10 @@ public function boot()
      */
     protected function registerRoutes()
     {
+        if (! Passport::$registersRoutes) {
+            return;
+        }
+
```

Comment:
Can you move the Route group inside the if statement here and negate the check?

Reply:
Done! @driesvints